### DISCUSSION TITLE
How to control text selection color

### DISCUSSION THREAD

**Discussion by kuriwaki** [mmistakes/minimal-mistakes#2765] [Q&A] [answered]
## Motivation

The default minimial-mistakes theme appears to do text selection in black background:
<img width="335" alt="image" src="https://user-images.githubusercontent.com/8290417/103158354-ca4c7780-478a-11eb-8ed9-e0d632dc9802.png">

## Suggestion

I'd like to revert this to the OS/browser default in my application, but I can't find find where to control this. Can you point to where to change this?

<img width="265" alt="image" src="https://user-images.githubusercontent.com/8290417/103158367-efd98100-478a-11eb-8ebf-88124879f5b4.png">

**Comment by mmistakes** [comment, marked as answer]
Edit this CSS to change the selection colors:

https://github.com/mmistakes/minimal-mistakes/blob/master/_sass/minimal-mistakes/_reset.scss#L33-L43

**Reply by gustavo-etal** [reply]
I did that but I do bundle exec jekyll serve and nothing changes, types are welcome
